.gitignore

```diff
@@ -31,7 +31,7 @@
 ################
 # setup.py working directory
 build
-
+venv
```

Comment:
A bit picky, but this probably should go in its own section as this is self defense against the behavior of niche workflow tools not standard Python tooling.